app/views/docs/databases.phtml

```diff
@@ -296,7 +294,7 @@ func main() async throws{
     </li>
 </ul>
 
-<p>The following operators are currently supported:</p>
+<p>The following query methods are currently supported:</p>
```

Comment:
I think this section is lacking. We need to better explain which attributes each method accepts and how the signature looks like. We should also mention each SDK provide helper classes for easy autocompletion in your IDE.